packages/mobile/src/components/atoms/SuspenseFallback/index.tsx

```diff
@@ -0,0 +1,17 @@
+import LoadingSpinner from "../LoadingSpinner";
+import $ from "./style.module.scss";
+
+type Props = {
+  height: string;
+};
+
+function SuspenseFallback(props: Props) {
+  const { height } = props;
+  return (
+    <div className={$["suspense-fallback"]} style={{ ...{ height } }}>
```

Comment:
여기도 `height` 하나라면 이렇게 써도 될 것 같아요!

```suggestion
    <div className={$["suspense-fallback"]} style={{ height }}>
```